tfe/resource_tfe_sentinel_policy.go

```diff
@@ -125,23 +133,22 @@ func resourceTFESentinelPolicyRead(d *schema.ResourceData, meta interface{}) err
 func resourceTFESentinelPolicyUpdate(d *schema.ResourceData, meta interface{}) error {
 	tfeClient := meta.(*tfe.Client)
 
-	if d.HasChange("enforce_mode") {
```

Comment:
Why did you remove this check?

Reply:
I'll add it back with the change for the comment above